app/src/components/pages/ScheduleManage.tsx

```diff
@@ -1,7 +1,202 @@
+    if (inputText !== '') {
+      const results = sampleSchedules.filter((schedule: ISchedule) => {
+        const matchText = schedule.term + ' ' + schedule.year;
+        return matchText.toLowerCase().includes(inputText.toLowerCase());
+      });
+      setFoundSchedules(results);
+    } else {
+      // Show all schedules
+      setFoundSchedules(sampleSchedules);
+    }
+    setSearch(inputText);
+  };
+
+  const getBgColor = (term: Term) => {
+    switch (term) {
+      case 'SUMMER':
+        return '#ff9800';
+      case 'WINTER':
+        return '#03a9f4';
+      case 'SPRING':
+        return '#ffc107';
+      default:
+        return '#ffc107';
+    }
+  };
+
+  const getIcon = (term: Term) => {
+    switch (term) {
+      case 'SUMMER':
+        return <WbSunnyIcon />;
+      case 'WINTER':
+        return <AcUnitIcon />;
+      case 'SPRING':
+        return <LocalFloristIcon />;
+      default:
+        return <CalendarMonthIcon />;
+    }
+  };
+
+  return (
+    <Box sx={{ flexGrow: 1, margin: 5 }}>
+      <Typography variant="h5" marginBottom={2}>
+        Manage Existing Schedules
+      </Typography>
+      <Grid container rowSpacing={3} alignItems={'center'}>
+        <Grid item>
+          <TextField
+            id="outlined-basic"
+            label="Search schedules..."
+            variant="outlined"
+            type="search"
+            value={search}
+            onChange={(e) => filter(e.target.value)}
+            className="input"
+            placeholder="Keyword"
+            sx={{ width: 350 }}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <List
+            sx={{
+              width: '100%',
+              bgcolor: 'background.paper',
+              position: 'relative',
+              overflow: 'auto',
+              maxHeight: 350,
+              '& ul': { padding: 0 }
+            }}
+            subheader={<li />}
+          >
+            {foundSchedules && foundSchedules.length > 0 ? (
+              foundSchedules.map((schedule) => (
+                <ListItem key={schedule.id}>
+                  <ListItemAvatar>
+                    <Avatar sx={{ bgcolor: getBgColor(schedule.term) }}>{getIcon(schedule.term)}</Avatar>
+                  </ListItemAvatar>
+                  <ListItemText primary={schedule.term} secondary={schedule.year} />
+                  <Button variant="outlined">View</Button>
+                  <Typography variant="body2" marginLeft={2}>
+                    Generated on {schedule.createdAt.toString()}
+                  </Typography>
+                </ListItem>
+              ))
+            ) : (
+              <Typography variant="subtitle1">No schedules found!</Typography>
+            )}
+          </List>
+        </Grid>
+
+        <Typography variant="h5" marginY={4}>
+          Generate New Schedule
+        </Typography>
+        <Grid container spacing={2} alignItems={'center'}>
+          <Grid item>
+            <TextField

```

Comment:
I believe our system will generate a whole year at once and will only generate for the current year. So I don't think a semester and year selector are neccessary.

Reply:
This is changing to allow for term/year so we do want to keep this